CRM/Core/BAO/Dashboard.php

```diff
@@ -357,11 +357,30 @@ public static function saveDashletChanges($columns, $contactID = NULL) {
       }
     }
 
-    // Disable inactive widgets
+    // Find dashlets in this domain.
+    $domainDashlets = civicrm_api3('Dashboard', 'get', [
+      'return' => array('id'),
+      'domain_id' => CRM_Core_Config::domainID(),
+    ]);
```

Comment:
@christianwach i think we may need to override the 25 limit here perhaps

Reply:
@seamuslee001 Ah yes, good catch. Fixed in cbf61ce366f87563621f581dd07b448d551ca014